packages/router/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remix-run/router",
-  "version": "1.15.3",
+  "version": "6.22.3",
```

Comment:
To keep things simple do we want to include this in lock-step with the other packages then?

Reply:
Yeah - I guess it could go into the changesets `fixed` config - but it’ll get deleted from there before we do any v7 pre-releases (along with native/compat/dom) I think so might not ever have a chance to do anything?